devel/namespaces/index.md

```diff
@@ -0,0 +1,167 @@
+---
+layout:   default
+class:    markdown
+---
+
+Namespaces
+==========
+
+TaskCluster has a number of namespaces defined to allow multiple users to get along without interfering with one another.
+The platform itself is agnostic to the structure of these namespaces, but infrastructure that interacts with the platform is dependent on the namespaces for security and correctness.
+
+This document necessarily contains information that is specific to users of the TaskCluster platform.
+As such, it is open to contributions from all users who wish to carve out a section of a namespace -- just [submit a pull request](https://github.com/taskcluster/taskcluster-docs).
+
+## Projects
+
+Most work at Mozilla falls into "projects", and these provide a nice organizational boundary for controlling access.
+We use a consistent name for each project in the various namespaces below -- something simple and without punctuation.
+The known projects are:
+
+ * `taskcluster` -- The taskcluster platform itself
+ * `releng` -- Mozilla release engineering (build, test, and release processes)
+ * `gaia` -- B2G/Firefox OS build system
+ * `bmo` -- bugzilla.mozilla.org development
+
+Please file a pull request to add your project!
+
+## Scopes
+
+Many scopes reflect the namespaces given elsewhere in this document, as described in the API documentation for the component.
+
+* `<component>:<verb>:` -
+   Scopes for most API actions have a prefix like this.
+   For example, the `queue.createTask` call is governed by a scope beginning with `queue:create-task:`.
+
+* `buildbot-bridge:`
```

Comment:
I'd like to put stuff that isn't part of TC core or platform into `project:<project>` so that we all know not to touch it, and who can legitimately touch it, without having to do a (manual) component -> owner mapping.  So `buildbot-bridge` and `signing` should both move into `project:releng:..`.